src/HelloPublisher/Program.cs

```diff
@@ -3,7 +3,7 @@
 using Azure.Messaging.EventGrid;
 
 var client1 = new EventGridPublisherClient(
-    new Uri("https://localhost:6000/webhook-200"),
+    new Uri("http://localhost:8080/comments-eg/api/events"),
```

Comment:
Doesn't work with https? I think I read in some MS doc that with EventGrid push delivery it will only push on https endpoint